keyboards/iris/keymaps/s1carii/rules.mk

```diff
@@ -0,0 +1,6 @@
+RGBLIGHT_ENABLE = no
+BACKLIGHT_ENABLE = no
+
+ifndef QUANTUM_DIR
```

Comment:
This block isn't needed

Reply:
The make fails without this block so I've replaced that for now. Unless I've misinterpreted what you mean here, is there another place that's referenced that I have to change as well? 

Thanks for the help here!